static/app/components/replays/player/scrubber.tsx

```diff
@@ -7,34 +8,70 @@ import * as Progress from 'sentry/components/replays/progress';
 import {useReplayContext} from 'sentry/components/replays/replayContext';
 import {divide, formatTime} from 'sentry/components/replays/utils';
 import {space} from 'sentry/styles/space';
+import toPercent from 'sentry/utils/number/toPercent';
+import useOrganization from 'sentry/utils/useOrganization';
 
 type Props = {
   className?: string;
 };
 
 function Scrubber({className}: Props) {
-  const {currentHoverTime, currentTime, replay, setCurrentTime} = useReplayContext();
-  const durationMs = replay?.getDurationMs();
+  const {currentHoverTime, currentTime, replay, setCurrentTime, timelineScale} =
+    useReplayContext();
+
+  const organization = useOrganization();
+  const hasNewTimeline = organization.features.includes('session-replay-new-timeline');
+
+  const durationMs = replay?.getDurationMs() ?? 0;
 
   const percentComplete = divide(currentTime, durationMs);
   const hoverPlace = divide(currentHoverTime || 0, durationMs);
 
+  const initialTranslate = 0.5 / timelineScale;
+
+  const starting = percentComplete < initialTranslate;
+  const ending = percentComplete + initialTranslate > 1;
+
+  const translate = () => {
+    if (starting) {
+      return 0;
+    }
+    if (ending) {
+      return 1 - 2 * initialTranslate;
+    }
+    return currentTime > durationMs ? 1 : percentComplete - initialTranslate;
+  };
+
   return (
     <Wrapper className={className}>
+      {className?.includes('PlayerScrubber') && hasNewTimeline ? (
```

Comment:
Ah you can't use the className for logic here as it will get compiled to a random string in production

Reply:
I would recommend passing in a prop to show the zoom indicators